chirp/drivers/ft7800.py

```diff
@@ -880,7 +895,7 @@ def get_features(self):
         rf = FTx800Radio.get_features(self)
         rf.has_sub_devices = self.VARIANT == ""
         rf.has_bank = True
-        rf.memory_bounds = (1, 500)
+        rf.memory_bounds = (1, 512)
```

Comment:
The extra memories are not normal channels right? They're specials, call channels, etc? If so, they should be implemented as such and not be presented as "channel 507" which won't match what the radio does. Look for some drivers with `valid_special_chans=` set for an example.

Reply:
Guess not everyone has every radio to test everything ;)